README.md

```diff
@@ -42,7 +42,7 @@ Clone the repository, change to the directory containing the repository.
 $ pip install -r requirements.txt
```

Comment:
The Django Girls one talks about (unsurprisingly) Django. I've seen the thing where people follow the entire tutorial even when they only need a small bit of it. https://tutorial.djangogirls.org/en/installation/